Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3.5
```

Comment:
Please add `MAINTAINER` field below:

```
MAINTAINER The XSnippet Team <dev@xsnippet.org>
```

Currently `dev@xsnippet.org` forwards all incoming mails to my personal one, but I'm going to reconfigure it to send it to you too.